settings.gradle

```diff
@@ -2,3 +2,4 @@ include ':paymentsheet-example'
 include ':stripe'
 include ':stripe-test-e2e'
 include ':example'
+include ':compose'
```

Comment:
I think there's probably a more descriptive name for this module. Will it be published? Used in the SDK? Simply a test bed?

Reply:
We agreed to make a module named paymentsheet.